src/app/utils/date-utils.ts

```diff
@@ -26,3 +26,7 @@ export function formatDuration(duration: moment.Duration): string {
         return moment.utc(durationMs).format('HH:mm');
     }
 }
+
+export function formatStartTime(duration: moment.Moment): string {
```

Comment:
I don't think that function helps us anyhow. It wraps just another function call. Also the name does not signify what's the output format of this time. Is that function only used in the pipe? If so, I'd simply move the entire

'    return moment(duration).format('HH:mm');'

into the pipe. Actually by looking at the pipe I also didn't know what is its output format. If the upper line was called straight in the pipe I could understand it quicker.

Happy to discuss though :D

Reply:
This function is also being used in calendar.component.ts - happy to import the pipe there instead of this function, if preferred?

Reply:
Yes please, you can instantiate pipe easily in components as well. That will keep it more unified

Reply:
Refactored